### ISSUE TITLE
Console error - W/Choreographer: Frame time is xxx ms in the future!

### ISSUE THREAD

**Issue by aryehof** [flutter/flutter#26635]
As requested by Chinmay Garde this issue contains details of a problem first reported on the Flutter Google Group.

[Click for item in Flutter Google Group](https://groups.google.com/forum/#!topic/flutter-dev/8CPnMWqhUtU)

## Issue

> I continually get multiple messages in the debug console when running any Flutter application (including the default "flutter create" application) that says something like the following ...

W/Choreographer(11099): Frame time is 0.047482 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.

[Click for example image](https://dl.dropboxusercontent.com/s/vjnlq887tjxce6r/2019-01-08_19-22-01.png)

Chinmay responded 

>That message is not being generated by either the Flutter engine or frameowork. Instead, it seems to be emanating from Android itself on your specific device. It seems benign but I would like to take a closer look. I haven't seen it on any Android devices I have on hand. **Could you please file an issue for this with the Android version and device you are seeing this message on?**

## Flutter Details
```
[√] Flutter (Channel dev, v1.1.5, on Microsoft Windows [Version 10.0.17134.523], locale en-US)
[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[√] Android Studio (version 3.2)
[√] VS Code, 64-bit edition (version 1.30.2)
[√] Connected device (1 available)

• No issues found!
```
## Android/Device Details

ASUS MeMo PAD HD 7
Model number: K00u
Android version: 4.2.2
Kernel version 3.4.5

**Comment by aryehof** [comment]
This issue was observed on a "rooted" android device.  Removing the "rooting" eliminated the error.  As the problem occurred on a non-standard configured device, I believe this item should be closed.

**Comment by hawkkiller** [comment]
It is not only on rooted devices

**Comment by Anas-jed** [comment]
It is not only on rooted devices (2)
I am also seeing this error, As far I think it is because of calling too many future await network requests in splash screen...

**Comment by larssn** [comment]
@aryehof Could you reopen?

**Comment by aryehof** [comment]
Reopened as requested.

**Comment by AkanshaJ07** [comment]
I am also seeing this same error. Could you please tell me how to resolve it?

**Comment by chinmaygarde** [comment]
I don't think there is anything Flutter can do to fix the log message itself as that is generated by the platform. But `0.047482` ms seems like a tolerably small number that it should not have noticeable impact on Flutter operation. We looked at the implementation of the Vsync waiter on Android couldn't spot any spot any issues with how it is wired up. Also, if the time is indeed in the future, the engine will make sure to defer frame scheduling till that point (notice the [`PostTaskForTime` here](https://github.com/flutter/engine/blob/2b42d53fd36587bfed5704b5a822e956ea6acedc/shell/common/vsync_waiter.cc#L140)). So even the engine does not depend on the target times being in the past.

Closing as working-as-intended from the perspective of frame scheduling and cant-fix from the logging perspective.
